### ISSUE TITLE
SlyCalendarView not showing

### ISSUE THREAD

**Issue by javierpe** [psinetron/slycalendarview#3]

**Comment by psinetron** [comment]
What happened? Errors?

**Comment by AndrewHossam** [comment]
No errors just nothing showed up

**Comment by victorjimbei** [comment]
This happens because init is not called from: 
SlyCalendarView(Context context, @Nullable AttributeSet attrs)
SlyCalendarView(Context context, @Nullable AttributeSet attrs, int defStyleAttr)

**Comment by Orlandroid** [comment]
yeah i have the same proble the view only show in dialog but in layout is not show
